test/e2e/banners.spec.ts

```diff
@@ -29,7 +29,8 @@ describe("error banners", () => {
     it("should show banner and disable button on /identifikation", () => {
       cy.login();
       cy.visit("/identifikation");
-      cy.url().should("include", "/identifikation");
+      cy.url().should("include", "/bundesIdent/primaryoption");
+      cy.contains("a", "Alle Identifikationsoptionen").click();
```

Comment:
I think this should be a separate test for `bundesIdent/primaryoption`. We should still keep the test for the `/identifikation`. For this we probably would need to add a click on the _Weiter_ button (`cy.contains("a", "Verstanden & weiter").click();`). Similar for the other tests in this file. Wdyt?

Reply:
but this test initially visits to `/identifikation` directly and by default, it will be redirected to  `/bundesident/primaryoption` option. Am I missing something?

Reply:
I still don't get where the `verstanden & weiter` button is. or maybe I don't see what you see 🤔 

These are the current spec. 
- Unleash is set
- I login
- I go to `/identifikation` 
- I get redirected to `/bundesIdent/primaryoption`
- I see a `Alle Identifikationsoptionen` button
- I click `Alle Identifikationsoptionen` button
- I see the Identifikation option page
- I see the button is disabled

Reply:
Thanks for again showing me the code. I somehow missed the `cy.contains("a", "Alle Identifikationsoptionen").click();` which is exactly what I meant with "Verstanden und weiter" (I think this is the button after login). I am so sorry for the back and forth.

So, we could still add tests for case a) but I would leave that up to you. Sorry for my misunderstanding and thanks for explaining it again 🙈

Reply:
No worries, we are though still testing the `a)` as you mentioned, we can see that explicitly because when user clicks `Alle Identifikationsoptionen`, the user goes to `/identifikation` route

```js
...
cy.contains("a", "Alle Identifikationsoptionen").click(); // => This takes user to /identifikation route

cy.get(bannerId).should("exist");
cy.contains("button", buttonLabel).should("be.disabled");
```

It would be better if we write something like this, I guess?
```js
...
cy.contains("a", "Alle Identifikationsoptionen").click(); // => This takes user to /identifikation route

cy.get(bannerId).should("exist");
cy.contains("button", buttonLabel).should("be.disabled");
```

Reply:
ah and you are right, some tests do not go to `/identifikation` route! I will adapt the changes then.

Reply:
Thank you. 
> It would be better if we write something like this, I guess?

I like this approach a lot 👍